apps/condo/domains/resident/tasks/sendBillingReceiptNotifications.task.js

```diff
@@ -2,10 +2,12 @@ const { createCronTask } = require('@condo/keystone/tasks')
 
 const { sendBillingReceiptNotifications } = require('./helpers')
 
-const WORK_DAYS_FROM = 10
-const WORK_DAYS_TO = 15
-const WEEKENDS_FROM = 12
-const WEEKENDS_TO = 15
+/** PODS work according to UTC, so time in cron-tasks should also be according to UTC too. */
+const UTC_TO_MSC = -3
```

Comment:
you can not change the scheduling time like so!

you must remove the old one by `removeCronTask` and add the new one by `createCronTask`

Reply:
fixed